src/app/leverage/constants.ts

```diff
@@ -36,6 +34,16 @@ export const btcLeverageTokens = [
 
 export const leverageTokens = [...ethLeverageTokens, ...btcLeverageTokens]
 
+export function getBaseTokens(chainId: number): Token[] {
+  if (chainId === BASE.chainId) return [ETH]
+  return [ETH, BTC]
+}
+
+export function getCurrencyTokens(chainId: number): Token[] {
+  if (chainId === BASE.chainId) return [ETH, WETH, USDC]
+  return [ETH, WETH, WBTC, USDC, USDT]
+}
```

Comment:
Changed into functions as Base has custom requirements - only ETH and WBTC and USDT are not on Base